app/models/admin_user.rb

```diff
@@ -1,6 +1,6 @@
 class AdminUser < ActiveRecord::Base
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
-  devise :database_authenticatable, 
-         :recoverable, :rememberable, :trackable, :validatable
+  devise :database_authenticatable, :rememberable
```

Comment:
It is `:rememberable`, from devise. It is for the "remember me" checkbox at login form and saving the token. See more here: https://github.com/plataformatec/devise